python/lsst/ctrl/bps/wms/htcondor/lssthtc.py

```diff
@@ -320,8 +319,11 @@ def htc_backup_files(wms_path, subdir=None, limit=100):
         subdir = Path(subdir)
         if subdir.is_absolute():
             if dest not in subdir.parents:
-                _LOG.warning("Invalid backup location: '%s' not in the submit directory, "
-                             "will use '%s' instead.", subdir, wms_path)
+                _LOG.warning(
+                    "Invalid backup location: '%s' not in the submit directory, " "will use '%s' instead.",
```

Comment:
Remove the spurious `" "` in the middle of the string. When black reformats it doesn't spot that it can do this.